### ISSUE TITLE
Junk in the cargo package source

### ISSUE THREAD

**Issue by bluss** [Geal/nom#67]
nom 0.3.10 has lots of large files in the cargo package source -- a 5 MB mp4 and more files olddoc, oldsrc (see at the end for file listing).

This is a reminder that cargo includes all non-ignored files in your working directory when you publish — look at `git status` before you publish.

(I downloaded all crates.io crates and I started to grep for junk)

```
.
├── bigbuckbunny.mp4
├── Cargo.toml
├── CHANGELOG.md
├── FullRecognition.jpg
├── gistfile1.txt
├── io.patch
├── LICENSE
├── links.txt
├── map.rs
├── mozilla-mp4.txt
├── mp4-layout.txt
├── nom
├── olddoc
│   ├── FiraSans-Medium.woff
│   ├── FiraSans-Regular.woff
│   ├── Heuristica-Italic.woff
│   ├── implementors
│   │   ├── core
│   │   │   ├── clone
│   │   │   │   └── trait.Clone.js
│   │   │   ├── cmp
│   │   │   │   ├── trait.Eq.js
│   │   │   │   └── trait.PartialEq.js
│   │   │   ├── fmt
│   │   │   │   └── trait.Debug.js
│   │   │   └── marker
│   │   │       └── trait.Copy.js
│   │   └── nom
│   │       ├── internal
│   │       │   ├── trait.GetInput.js
│   │       │   └── trait.GetOutput.js
│   │       ├── producer
│   │       │   └── trait.Producer.js
│   │       └── util
│   │           ├── trait.AsBytes.js
│   │           └── trait.HexDisplay.js
│   ├── jquery.js
│   ├── main.css
│   ├── main.js
│   ├── nom
│   │   ├── consumer
│   │   │   ├── enum.ConsumerState.html
│   │   │   ├── index.html
│   │   │   ├── sidebar-items.js
│   │   │   └── trait.Consumer.html
│   │   ├── index.html
│   │   ├── internal
│   │   │   ├── enum.Err.html
│   │   │   ├── enum.IResult.html
│   │   │   ├── enum.Needed.html
│   │   │   ├── index.html
│   │   │   ├── sidebar-items.js
│   │   │   ├── trait.GetInput.html
│   │   │   └── trait.GetOutput.html
│   │   ├── macro.alt!.html
│   │   ├── macro.alt_parser!.html
│   │   ├── macro.apply!.html
│   │   ├── macro.call!.html
│   │   ├── macro.chain!.html
│   │   ├── macro.chaining_parser!.html
│   │   ├── macro.closure!.html
│   │   ├── macro.cond!.html
│   │   ├── macro.count!.html
│   │   ├── macro.dbg_dmp!.html
│   │   ├── macro.dbg!.html
│   │   ├── macro.delimited!.html
│   │   ├── macro.error!.html
│   │   ├── macro.filter!.html
│   │   ├── macro.flat_map!.html
│   │   ├── macro.is_a!.html
│   │   ├── macro.is_not!.html
│   │   ├── macro.length_value!.html
│   │   ├── macro.many0!.html
│   │   ├── macro.many1!.html
│   │   ├── macro.map!.html
│   │   ├── macro.map_opt!.html
│   │   ├── macro.map_res!.html
│   │   ├── macro.named!.html
│   │   ├── macro.opt!.html
│   │   ├── macro.pair!.html
│   │   ├── macro.peek!.html
│   │   ├── macro.preceded!.html
│   │   ├── macro.pusher!.html
│   │   ├── macros
│   │   │   ├── index.html
│   │   │   └── sidebar-items.js
│   │   ├── macro.separated_list!.html
│   │   ├── macro.separated_nonempty_list!.html
│   │   ├── macro.separated_pair!.html
│   │   ├── macro.tag!.html
│   │   ├── macro.take!.html
│   │   ├── macro.take_str!.html
│   │   ├── macro.take_until_and_consume!.html
│   │   ├── macro.take_until_either_and_consume!.html
│   │   ├── macro.take_until_either!.html
│   │   ├── macro.take_until!.html
│   │   ├── macro.terminated!.html
│   │   ├── nom
│   │   │   ├── fn.alpha.html
│   │   │   ├── fn.alphanumeric.html
│   │   │   ├── fn.be_f32.html
│   │   │   ├── fn.be_f64.html
│   │   │   ├── fn.begin.html
│   │   │   ├── fn.be_i16.html
│   │   │   ├── fn.be_i32.html
│   │   │   ├── fn.be_i64.html
│   │   │   ├── fn.be_i8.html
│   │   │   ├── fn.be_u16.html
│   │   │   ├── fn.be_u32.html
│   │   │   ├── fn.be_u64.html
│   │   │   ├── fn.be_u8.html
│   │   │   ├── fn.digit.html
│   │   │   ├── fn.eof.html
│   │   │   ├── fn.is_alphabetic.html
│   │   │   ├── fn.is_alphanumeric.html
│   │   │   ├── fn.is_digit.html
│   │   │   ├── fn.is_space.html
│   │   │   ├── fn.le_i16.html
│   │   │   ├── fn.le_i32.html
│   │   │   ├── fn.le_i64.html
│   │   │   ├── fn.le_i8.html
│   │   │   ├── fn.length_value.html
│   │   │   ├── fn.le_u16.html
│   │   │   ├── fn.le_u32.html
│   │   │   ├── fn.le_u64.html
│   │   │   ├── fn.le_u8.html
│   │   │   ├── fn.line_ending.html
│   │   │   ├── fn.multispace.html
│   │   │   ├── fn.not_line_ending.html
│   │   │   ├── fn.print.html
│   │   │   ├── fn.sized_buffer.html
│   │   │   ├── fn.space.html
│   │   │   ├── fn.tag_cl.html
│   │   │   ├── index.html
│   │   │   └── sidebar-items.js
│   │   ├── producer
│   │   │   ├── enum.ProducerState.html
│   │   │   ├── index.html
│   │   │   ├── sidebar-items.js
│   │   │   ├── struct.FileProducer.html
│   │   │   ├── struct.MemProducer.html
│   │   │   └── trait.Producer.html
│   │   ├── sidebar-items.js
│   │   └── util
│   │       ├── enum.ErrorCode.html
│   │       ├── fn.add_error_pattern.html
│   │       ├── fn.code_from_offset.html
│   │       ├── fn.compare_error_paths.html
│   │       ├── fn.error_to_list.html
│   │       ├── fn.generate_colors.html
│   │       ├── fn.prepare_errors.html
│   │       ├── fn.print_codes.html
│   │       ├── fn.print_error.html
│   │       ├── fn.print_offsets.html
│   │       ├── fn.reset_color.html
│   │       ├── fn.slice_to_offsets.html
│   │       ├── fn.write_color.html
│   │       ├── index.html
│   │       ├── sidebar-items.js
│   │       ├── trait.AsBytes.html
│   │       └── trait.HexDisplay.html
│   ├── normalize.css
│   ├── playpen.js
│   ├── search-index.js
│   ├── SourceCodePro-Regular.woff
│   ├── SourceCodePro-Semibold.woff
│   ├── SourceSerifPro-Bold.woff
│   ├── SourceSerifPro-Regular.woff
│   └── src
│       └── nom
│           ├── consumer.rs.html
│           ├── internal.rs.html
│           ├── lib.rs.html
│           ├── macros.rs.html
│           ├── nom.rs.html
│           ├── producer.rs.html
│           └── util.rs.html
├── oldsrc
│   ├── conveyor.rs
│   └── main.rs
├── README.md
├── realworld
│   └── dropbox.txt
├── small.mp4
├── src
│   ├── accumulator.rs
│   ├── bits.rs
│   ├── bytes.rs
│   ├── consumer.rs
│   ├── generator.rs
│   ├── internal.rs
│   ├── lib.rs
│   ├── macros.rs
│   ├── nom.rs
│   ├── producer.rs
│   └── util.rs
├── tests
│   ├── arithmetic.rs
│   ├── ini.rs
│   ├── issues.rs
│   ├── mp4.rs
│   ├── omnom.rs
│   └── test1.rs
├── thoughts.txt
└── :w

24 directories, 181 files
```

**Comment by Geal** [comment]
Good catch, thanks! I'll publish another package today. I did not know about that cargo "feature" :)

**Comment by Geal** [comment]
Fixed in 7e225f20d622. I just published the version 0.3.11.

Thank you for your help!

**Comment by bluss** [comment]
Great!
